src/main/java/org/sonarsource/sonarlint/ls/EnginesFactory.java

```diff
@@ -60,6 +58,7 @@ public StandaloneSonarLintEngine createStandaloneEngine() {
     try {
       StandaloneGlobalConfiguration configuration = StandaloneGlobalConfiguration.builder()
         .setExtraProperties(prepareExtraProps())
+        .addEnabledLanguages(Language.HTML, Language.JAVA, Language.JS, Language.PHP, Language.PYTHON, Language.TS)
```

Comment:
Nitpick: I would make it explicit that the languages enabled in connected mode are an extension of those enabled in standalone. See e.g [how it's declared in SLI](https://github.com/SonarSource/sonarlint-intellij/blob/a4f779ee7ca39d064bf2418011fbdfe0db62de98/src/main/java/org/sonarlint/intellij/core/SonarLintEngineFactory.java#L50)

Reply:
Done.